core/src/main/java/org/hisp/dhis/android/core/analytics/trackerlinelist/internal/evaluator/TrackerLineListSQLLabel.kt

```diff
@@ -33,5 +33,8 @@ import org.hisp.dhis.android.core.program.programindicatorengine.internal.Progra
 internal object TrackerLineListSQLLabel {
     const val EventAlias = ProgramIndicatorSQLUtils.event
     const val EnrollmentAlias = ProgramIndicatorSQLUtils.enrollment
+    const val TrackedEntityInstanceAlias = "teiAlias"
+    const val programStageAlias = "programStageAlias"
     const val OrgunitAlias = "ou"
+    const val subOrgunitAlias = "subou"
```

Comment:
Maybe capital letter for all of them? Looking at Kotlin naming conventions, it seems we should use uppercase, but I feel it will make the code less readable. https://kotlinlang.org/docs/coding-conventions.html#property-names.

At least, follow the same formatting for all the variables in the block

Reply:
It seems there is some inconsistent aliases in the repo. There are some abbreviations like "ou" or "tei" that are in lower case and others like "CCOL" in upper case. Should these also be normalized?

Reply:
sorry, I meant the variable name, not the value

Reply:
Understood, fixed in the last commit.